src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsLambda/HandlerMethodWrapper.cs

```diff
@@ -224,10 +230,19 @@ void InvokeTryProcessResponse(Task responseTask)
                     {
                         return;
                     }
+
+                    if (responseTask.Status == TaskStatus.Faulted)
+                    {
+                        transaction.NoticeError(responseTask.Exception);
+                    }
```

Comment:
Should this be moved before the previous if statement so that we notice the error before the early return?